src/test/java/net/minecraftforge/debug/DynBucketTest.java

```diff
@@ -180,6 +162,29 @@ public void onBucketFill(FillBucketEvent event)
         }
     }
 
+    @Mod.EventBusSubscriber(value = Side.CLIENT, modid = MODID)
+    public static class ClientEventHandler
```

Comment:
I considered just going with `@SideOnly` but that
1) Still would have crashed with the `ItemTESR` and `DebugModel` class
2) Would be against my desire to make test mods use best practices as much as possible and use of SideOnly is strongly discouraged for mods